prov/verbs/src/ep_rdm/verbs_rdm.h

```diff
@@ -110,10 +110,10 @@ do {                                                                	\
 	 PEND_SEND_IS_LIMITED(_ep))
 
 struct fi_ibv_rdm_header {
-	uint64_t imm_data;          // TODO: not implemented
+/*	uint64_t imm_data; TODO: not implemented */
```

Comment:
In this meaning, if I correctly understand, wire protocol should be changed soon anyway with reusing headers from fi_proto.h support shm, right? I don't think that imm support will be added earlier without explicit request.